### PR TITLE
Add previous state and styling to logger

### PR DESCRIPTION
**Pull request by griffinmichl**
I think these minor changes will lead to a much nicer DX. Screenshot below shows what it looks like with the changes.

screen shot 2016-08-29 at 8 12 22 pm

### CODE REVIEW

**Comment by phil303** [comment]
Hey Griffin! Thanks for the patch and welcome to the team!

I think overall this is a welcome change. My one concern is that it doesn't seem like `console.group` is [well supported](https://developer.mozilla.org/en-US/docs/Web/API/Console/group) on mobile browsers. While we do most of our dev work on our laptops, we do sometimes run dev code on mobile browsers so I believe this is going to cause a problem in those cases.

Also, as just an fyi and not a "use this instead" (I think this is still a good patch), have you tried the [redux dev tools extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en)? It's an amazing tool. Once installed, we [programmatically stop printing](https://github.com/reddit/node-platform/blob/master/lib/modules/Client.js#L28) to the console, which leaves it open for more specific print statements.

**Comment by griffinmichl** [comment]
Good point! Mobile browsers skipped my mind.

I use and love redux dev tools. Just figured if we're using logger at all, it might as well be easy to read.

I amended the commit to check for console.group before using it.

**Comment by griffinmichl** [comment]
And thanks for the welcome! I don't think we've met yet. Looking forward to getting to know everyone.

File: lib/modules/Logger.js [hunk 1]
```diff
@@ -1,6 +1,17 @@
 export default store => next => action => {
-  console.log('dispatching', action);
-  const result = next(action);
-  console.log('next state', store.getState());
+  let result;
+  console.group(action.type);
```
Comment (nramadas) [inline lib/modules/Logger.js]: welcome to the team! also, should this fall within the `if (console.group)` clause?

File: lib/modules/Logger.js [hunk 2]
```diff
@@ -1,6 +1,17 @@
 export default store => next => action => {
-  console.log('dispatching', action);
-  const result = next(action);
-  console.log('next state', store.getState());
+  let result;
+  console.log(action.type);
+  if (console.group) {
```
Comment (nramadas) [inline lib/modules/Logger.js:3]: this looks good! I think you're missing the initial `console.group` call though. how about something like:

``` es6
let result;
if (console.group) {
  console.group(action.type);
  /* yadda yadda yadda */
  console.groupEnd();
} else {
  /* normal output */
}
```

**Comment by nramadas** [comment]
:nail_care:

**Comment by griffinmichl** [comment]
💇

**Comment by nramadas** [comment]
:fish:

**Pull request closed without merging**